### DISCUSSION TITLE
Categories και tags για παραδοτέο Α1

### DISCUSSION THREAD

**Discussion by mariosconsta** [courses-ionio/hci#1662] [Announcements]
Τι βάζουμε στα categories και στα tags για το παραδοτέο Α1? Για categories πρέπει να είναι τα Κεφάλαια που έχει [εδώ ](https://pibook.epidro.me/chapters)και για τα tags το ίδιο όπως έχει [εδώ](https://pibook.epidro.me/tags/); Η είναι δικά μας που σχετίζονται με την φωτογραφία;

**Comment by FaidonTsetsis** [comment]
Στις κατηγορίες θα βάλεις αυτές του βιβλίου ανάλογα με το θέμα της εικόνας σου, πχ αρχέτυπα κτλ. Τα tags πρέπει να είναι σχετικά με την ιδέα πίσω από την εικόνα που θα προσθέσεις, δηλαδή κάτι σχετικό με ό,τι γράφεις στη λεζάντα.

**Reply by epidrome** [reply]
> παρατηρούμε τα δεδομένα στις εικόνες-λεζάντες-ετικέτες-κατηγορίες-χρονολογία, συνήθως είναι από διαδραστικά συστήματα 10+ χρόνια παλιότερα, και συνδέονται απευθείας με το περιεχόμενο του βιβλίου: Οι κατηγορίες είναι κεφάλαια-υποκεφάλαια, οι ετικέτες είναι εταιρείες, άνθρωποι, σημαντικά προϊόντα, ενώ οι λεζάντες δίνουν έμφαση στην διάδραση ή/και στο λογισμικό-υλικό διάδρασης και όχι μόνο στην τεχνολογία ή μόνο στον άνθρωπο.
